### ISSUE TITLE
Blitz more Libz!

### ISSUE THREAD

**Issue by KodrAus** [rust-lang-nursery/ecosystem-wg#11]
The [libz blitz](https://blog.rust-lang.org/2017/05/05/libz-blitz.html) offers a great framework for exploring, evaluating and polishing libraries. Should we do more evaluations for stabilising libraries this year? If so, what libraries do folks think could benefit from a blitz?

**Comment by skade** [comment]
Yes. Please note there was a plea for a better name last year.

**Comment by KodrAus** [comment]
> Please note there was the wish for a better name last year.

Ah I wasn't aware of that, and can see why some folks might prefer a different name. What if we just call them 'evaluations'?

**Comment by skade** [comment]
Let's bikeshed a little on that, catchy would be desirable.

**Comment by Keats** [comment]
How about making https://github.com/Gilnaa/globwalk the next https://github.com/rust-lang-nursery/glob ?
That would accomplish some of https://github.com/rust-lang-nursery/glob/issues/59 provided all the other issues can be fixed.

**Comment by epage** [comment]
As I brought up in the CLI WG, what about helping to organize libz blitz centered around each of the other WGs.  For example, the CLI WG could identify features we need, crates that fulfill those features, and then work on a libz blitz to help get them fleshed out.

**Comment by Enet4** [comment]
I would like to bring `chrono` to this table, although I cannot be sure how the group can intervene. https://github.com/chronotope/chrono/issues/195 Although the library is in a fairly working state right now, multiple breaking changes are desired before a stable release, and the current maintainer is lacking the time to fulfil these ideas. Considering also that there are over 500 direct dependents, `diesel` being one of them, it seems to me like a project worth catalysing.

**Comment by KodrAus** [comment]
@Enet4 `chrono` has been on my mind for a while. @quodlibetor might have some thoughts on where/how contributions could be the most helpful.

@epage Using the domain working groups as a source for libraries to focus on sounds good to me! Particularly libraries that are common between use-cases in multiple domains. Maybe we should start a thread on one of the forums to collect a set?

@Keats Should we suggest `globwalk` on https://github.com/rust-lang-nursery/glob/issues/59? We probably need to try revive some interest in that issue.

@skade I've opened #15 to work on a new name, so we can keep the threads a bit separate.

**Comment by quodlibetor** [comment]
Yes, I've been thinking about improving chrono's API for a _long_ time. I've been intending to write up my plans, as well, perhaps blitzing would give me the kick I need to actually do it? It'd be welcome, anyway.

**Comment by Keats** [comment]
@KodrAus the `globwalk` crate came from https://github.com/BurntSushi/ripgrep/pull/765 so I believe we just need to improve on it first to make sure it satisfies all the requirements of the current `glob` crate

**Comment by Dylan-DPC** [comment]
@quodlibetor let me know if you need help with `chrono`

**Comment by quodlibetor** [comment]
Until we get an official ~blitz~ going for chrono I've written up some notes about some of the most egregious things I think need to be iterated upon: https://github.com/chronotope/chrono-rfcs/blob/master/notes/api-issues.md

I've spent a bit of time switching some types around and redesigning some traits and I think we can get to a much simpler API while still maintaining all of chrono's current power.
